other_processing/0.download-profiles-from-figshare/download.py

```diff
@@ -10,24 +10,25 @@
 In the pipeline, we measured several morphology features for every single cell.
 We then aggregated the output of CellProfiler using cytominer-database.
 Cytominer-database compiles all single cell profiles into a single `sqlite` database.
-We uploaded the `sqlite` files to NIH Figshare 
+We uploaded the `sqlite` files to NIH Figshare
 https://nih.figshare.com/articles/dataset/Cell_Health_-_Cell_Painting_Single_Cell_Profiles/9995672/1.
 These data are publicly available
```

Comment:
Slight rewording suggestion:

```suggestion
We uploaded the `sqlite` files to NIH Figshare, where they are publicly available:
https://nih.figshare.com/articles/dataset/Cell_Health_-_Cell_Painting_Single_Cell_Profiles/9995672/1.
```